src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -179,6 +181,13 @@ class ActivityFeed extends React.Component<Props, State> {
         }
     };
 
+    handleToggleHeaderShadow = () => {
```

Comment:
Can we rename this to `handleFeedScroll` to make it less specific to the underlying logic?